lib/common/router.go

```diff
@@ -2,13 +2,23 @@ package common
 
 import (
 	"net/http"
+	"strings"
 
 	"github.com/gorilla/mux"
 )
 
 func PostAndJSONMatcher(r *http.Request, rm *mux.RouteMatch) bool {
 	if r.Method == "POST" {
-		if r.Header.Get("Content-Type") != "application/json" {
+		ct := r.Header.Get("Content-Type")
+		if len(strings.TrimSpace(ct)) < 1 {
+			return false
+		}
+		spl := strings.SplitN(ct, ";", 2)
+		if len(spl) == 2 {
```

Comment:
`Content-Type` can have any number of parameters (https://www.w3.org/Protocols/rfc1341/4_Content-Type.html) so we should just take the first entry, always.

Reply:
Yes, you correct.